### PR TITLE
Fix ansible-test coverage traceback.

### LINKED ISSUE #71443: ansible-test coverage leads to 'Namespace' object has no attribute 'remote_endpoint'
##### SUMMARY

We are running `ansible-test coverage` from the devel branch of Ansible on the `community.kubernetes` collection. It seems like sometime between yesterday morning and today, something in Ansible changed that is causing the error:

```
'Namespace' object has no attribute 'remote_endpoint'
```

See: https://github.com/ansible-collections/community.kubernetes/runs/1025104583?check_suite_focus=true#step:6:17

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
ansible-test

##### ANSIBLE VERSION

```paste below
devel (installed via pip)
```

##### CONFIGURATION

```paste below
N/A
```

##### OS / ENVIRONMENT

Ubuntu 20.04 (GitHub Actions CI)

##### STEPS TO REPRODUCE

```
# Inside the collection.kubernetes directory:
pip install https://github.com/ansible/ansible/archive/devel.tar.gz
ansible-test integration --docker -v --color --retry-on-error --python 3.6 --continue-on-error --diff --coverage
ansible-test coverage xml -v --requirements --group-by command --group-by version
```

##### EXPECTED RESULTS

Coverage report is generated.

##### ACTUAL RESULTS

```
Run ansible-test coverage xml -v --requirements --group-by command --group-by version
  ansible-test coverage xml -v --requirements --group-by command --group-by version
  shell: /bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.6.11/x64
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.6.11/x64/bin/ansible-test", line 28, in <module>
    main()
  File "/opt/hostedtoolcache/Python/3.6.11/x64/bin/ansible-test", line 24, in main
    cli_main()
  File "/opt/hostedtoolcache/Python/3.6.11/x64/lib/python3.6/site-packages/ansible_test/_internal/cli.py", line 159, in main
    config = args.config(args)  # type: CommonConfig
  File "/opt/hostedtoolcache/Python/3.6.11/x64/lib/python3.6/site-packages/ansible_test/_internal/coverage/__init__.py", line 59, in __init__
    super(CoverageConfig, self).__init__(args, 'coverage')
  File "/opt/hostedtoolcache/Python/3.6.11/x64/lib/python3.6/site-packages/ansible_test/_internal/config.py", line 99, in __init__
    self.remote_endpoint = args.remote_endpoint  # type: t.Optional[str]
AttributeError: 'Namespace' object has no attribute 'remote_endpoint'
```

### PR DESCRIPTION
**Pull request by mattclay**
##### SUMMARY

https://github.com/ansible/ansible/pull/71413 introduced a traceback when running `ansible-test coverage`.

This PR fixes the traceback and adds minimal tests for `ansible-test coverage`.

Resolves https://github.com/ansible/ansible/issues/71443

##### ISSUE TYPE

Bugfix Pull Request

##### COMPONENT NAME

ansible-test

### CODE REVIEW

File: changelogs/fragments/ansible-test-coverage-py26.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test - Fix ``ansible-test coverage`` reporting sub-commands (``report``, ``html``, ``xml``) on Python 2.6.
```
Comment (felixfontein) [inline changelogs/fragments/ansible-test-coverage-py26.yml:2]: How is this related to Python 2.6? Or do you mean 3.6?

Comment (mattclay) [reply changelogs/fragments/ansible-test-coverage-py26.yml:2]: The new tests uncovered a bug running the coverage reporting commands on Python 2.6. Since that bug exists in released versions of ansible-test I added a changelog fragment for the fix which is included in this PR. I didn't add a changelog fragment for the coverage traceback since it hasn't gone into any release.

Comment (felixfontein) [reply changelogs/fragments/ansible-test-coverage-py26.yml:2]: Ah, ok, makes sense :)
